File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Layout.Events;
using DevExpress.XtraGrid.Views.Layout;
using DevExpress.XtraLayout.Utils;
using DevExpress.XtraEditors;

namespace DXSample {
    public partial class Main : XtraForm {
        public Main() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            recordBindingSource.DataSource = DataHelper.GetData(10);
        }
        private void layoutView1_CustomCardLayout(object sender, LayoutViewCustomCardLayoutEventArgs e) {
            LayoutView view = sender as LayoutView;
            if(e.RowHandle == view.FocusedRowHandle) {
                e.CardDifferences.AddItemDifference("Group1", LayoutItemDifferenceType.GroupExpanded, true);
            }
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Layout.Events
Imports DevExpress.XtraGrid.Views.Layout
Imports DevExpress.XtraLayout.Utils
Imports DevExpress.XtraEditors

Namespace DXSample
	Partial Public Class Main
		Inherits XtraForm

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
			recordBindingSource.DataSource = DataHelper.GetData(10)
		End Sub
		Private Sub layoutView1_CustomCardLayout(ByVal sender As Object, ByVal e As LayoutViewCustomCardLayoutEventArgs) Handles layoutView1.CustomCardLayout
			Dim view As LayoutView = TryCast(sender, LayoutView)
			If e.RowHandle = view.FocusedRowHandle Then
				e.CardDifferences.AddItemDifference("Group1", LayoutItemDifferenceType.GroupExpanded, True)
			End If
		End Sub
	End Class
End Namespace